tests/asm.arm/Cargo.toml

```diff
@@ -0,0 +1,6 @@
+[package]
```

Comment:
Could you also add an arm64 test? I think people might care about that more than 32-bit.

Reply:
Will do.

Reply:
Done.